pwa-devdocs/src/venia-pwa-concept/project-structure/index.md

```diff
@@ -40,19 +40,19 @@ The `venia-static` directory contains the `favicon.ico` icon file, `icons` folde
 
 The `src` directory contains the PWA source code for the Venia theme, which are split into functional subdirectories.
 
-### `src/index.js`
+### [`src/index.js`][]
 
 The `src/index.js` file is the entry point of Venia.
 It sets the ApolloProvider, the ReduxStore, and the Router configuration and App Component.
 
-### `src/store.js`
+### [`src/store.js`][]
 
 The `src/store.js` file is responsible for creating a Redux store.
 It also combines Redux reducers and middlewares.
 
-### `src/sw.js`
+### [`src/registerSW.js`][]
```

Comment:
I don't think this is accurate. `registerSw.js` contains logic to register a SW on the app. `sw.js` contains the main Service Worker logic. `sw.js` is the first file from where the webpack bundling of the SW starts.